task-3/3.go

```diff
@@ -0,0 +1,19 @@
+package main
+
+import (
+	"fmt"
+)
+
+func main() {
+	var n int
+	fmt.Scanf("%d", &n)
+	var arr = make([]int, n)
```

Comment:
когда ты сразу инициализируешь переменную, а ```make``` именно это и делает, по кодстайлу принято писать так
```suggestion
	arr := make([]int, n)
```